internal/global/common/custom_event_triggerable.go

```diff
@@ -0,0 +1,5 @@
+package common
+
+type CustomEventTriggerable interface {
+	TriggerEvent()
+}
```

Comment:
this should be specific to trigger dots/element modifiers and not some generic "any custom event" trigger